spec/models/content_item_spec.rb

```diff
@@ -614,5 +614,34 @@
         end
       end
     end
+
+    describe 'phase' do
```

Comment:
This block doesn't really belong here (it's in the "linked items" describe block").  It would be better in the "validations" block at the top of the file.